README.md

```diff
@@ -44,6 +44,82 @@ docker build -t pca-assessment .
 We welcome contributions!  Please see [here](CONTRIBUTING.md) for
 details.
 
+## Assessment JSON Field Dictionary ##
+
+The following items are included in the assessment JSON as produced by the `pca-wizard`.
+An example assessment JSON can be found [here](src/assessment/sample_assessment.json).
+
+| Name | Description | Type | Default | Required |
+|------|-------------|:----:|:-------:|:--------:|
+| id | RV number. | string | | yes |
```

Comment:
Instead of "RV number" here, I'd prefer to see something like "Assessment identifier (e.g. "RV0000"), since "RV number" is somewhat meaningless outside of our world.  Remember that our goal is to create tools that are usable by anyone, not just the folks on our team.